### PR TITLE
SPS location moved back to centerline

### PR DESCRIPTION
**Pull request by indy91**

### CODE REVIEW

File: Orbitersdk/samples/ProjectApollo/src_csm/saturn.cpp [hunk 1]
```diff
@@ -4750,6 +4757,97 @@ void Saturn::StartSeparationPyros()
...
+	if (propmass > MSUMP)
+	{
+		sumpm = MSUMP;
+		storem = propmass - MSUMP;
+	}
+	else
+	{
+		sumpm = propmass;
+		storem = 0;
+	}
```
Comment (ThymoNL) [inline Orbitersdk/samples/ProjectApollo/src_csm/saturn.cpp:4811]: Let's be consistent with the way we put braces

```suggestion
	if (propmass > MSUMP) {
		sumpm = MSUMP;
		storem = propmass - MSUMP;
	} else {
		sumpm = propmass;
		storem = 0;
	}
```

File: Orbitersdk/samples/ProjectApollo/src_csm/saturn.cpp [hunk 2]
```diff
@@ -4750,6 +4757,97 @@ void Saturn::StartSeparationPyros()
...
+
+	oxstorem = storem * 1.6 / (1.0 + 1.6);
+	fuelstorem = storem * 1.0 / (1.0 + 1.6);
+	oxsumpm = sumpm * 1.6 / (1.0 + 1.6);
+	fuelsumpm = sumpm * 1.0 / (1.0 + 1.6);
+
+	static const double oxid_store_tank_param[3] = { -8.385141e-9, 0.0061750118, 838.7809363};
```
Comment (ThymoNL) [inline Orbitersdk/samples/ProjectApollo/src_csm/saturn.cpp]:
```suggestion
	static const double oxid_store_tank_param[3] = { -8.385141e-9, 0.0061750118, 838.7809363 };
```

**Review by rcflyinghokie — DISMISSED** [review]
Tested with TEI both with and without new N48 angles. Other than the expected transient, no ill effects noted.

File: Orbitersdk/samples/ProjectApollo/src_csm/saturn.cpp [hunk 3]
```diff
@@ -4750,6 +4757,97 @@ void Saturn::StartSeparationPyros()
...
+
+	COG = (CGCM*MCM + CGSM * MSM + ox_store_CG * oxstorem + ox_sump_CG * oxsumpm + fuel_store_CG * fuelstorem + fuel_sump_CG * fuelsumpm) / mass;
+	//sprintf(oapiDebugString(), "%lf %lf %lf %lf", propmass, COG.x, COG.y, COG.z);
+	COG = COG * 0.0254;
+
+	//Convert to Orbiter
+	COG = _V(COG.y, -COG.z, COG.x - 24.538);
+
+	//PMI
+	//PMI = _V(4.3972, 4.6879, 1.6220);
```
Comment (ThymoNL) [inline Orbitersdk/samples/ProjectApollo/src_csm/saturn.cpp]: I think this is no longer used?

```suggestion
```

**Review by ThymoNL — CHANGES_REQUESTED** [review]
Couple minor things, rest looks good.

**Review by ThymoNL — APPROVED** [review]

**Pull request merged**